### PR TITLE
Fix to allow Ofx or Omon volcello data in recipe.py

### PR DESCRIPTION
**Pull request by ledm**
**Tasks**

-   [x] [Create an issue](https://github.com/ESMValGroup/ESMValCore/issues) to discuss what you are going to do, if you haven't done so already (and add the link at the bottom)
-   [x] This pull request has a descriptive title that can be used in a changelog
-   [ ] Add unit tests
-   [x] Public functions should have a numpy-style docstring so they appear properly in the [API documentation](https://esmvaltool.readthedocs.io/projects/esmvalcore/en/latest/api/esmvalcore.html). For all other functions a one line docstring is sufficient.
-   [ ] If writing a new/modified preprocessor function, please update the [documentation](https://esmvaltool.readthedocs.io/projects/esmvalcore/en/latest/esmvalcore/preprocessor.html)
-   [ ] Circle/CI tests pass. Status can be seen below your pull request. If the tests are failing, click the link to find out why.
-   [ ] Codacy code quality checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why. If you suspect Codacy may be wrong, please ask by commenting.
-   [ ] Please use `yamllint` to check that your YAML files do not contain mistakes
-   [ ] If you make backward incompatible changes to the recipe format, make a new pull request in the [ESMValTool repository](https://github.com/ESMValGroup/ESMValTool) and add the link below

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes {https://github.com/ESMValGroup/ESMValCore/issues/405}

### CODE REVIEW

**Comment by ledm** [comment]
I guess @valeriupredoi shouldn't be a reviewer, as he wrote the code we're changing.

**Comment by valeriupredoi** [comment]
yes but @predoivaleriu can :grin:
@mattiarighi -> @ledm has tested this :beer:

File: esmvalcore/_recipe.py [hunk 1]
```diff
@@ -401,11 +401,20 @@ def _get_correct_fx_file(variable, fx_varname, config_user):
...
         raise RecipeError(
             f"Project {var['project']} not supported with fx variables")
 
     fx_files = _get_input_files(fx_var, config_user)
+
+    # If can't find Ofx files, then try with original variable mip
+    if not fx_files and fx_var['mip'] == 'Ofx':
+        fx_var['mip'] = var['mip']
+        fx_files = _get_input_files(fx_var, config_user)
+
```
Comment (valeriupredoi) [inline esmvalcore/_recipe.py]: maybe add a message here `logger.info("Tried looking for both Ofx and %s", var['mip'])`

Comment (ledm) [reply esmvalcore/_recipe.py]: Done.

**Comment by valeriupredoi** [comment]
related (but not functionally per se) to https://github.com/ESMValGroup/ESMValCore/pull/432 - I reckon we should get this one in first then #432 immediately after :beer:

**Comment by schlunma** [comment]
superseded by #432

**Pull request closed without merging**